lib/MusicBrainz/Server/Entity/URL/Bugs.pm

```diff
@@ -0,0 +1,23 @@
+package MusicBrainz::Server::Entity::URL::Bugs;
+
+use Moose;
+use utf8;
+
+extends 'MusicBrainz::Server::Entity::URL';
+with 'MusicBrainz::Server::Entity::URL::Sidebar';
+
+sub sidebar_name { 'Bugs!' }
+
+__PACKAGE__->meta->make_immutable;
+no Moose;
+1;
+
+=head1 COPYRIGHT AND LICENSE
+
+Copyright (C) 2022 MetaBrainz Foundation
```

Comment:
It is fine but since it is your first contribution I have to ask you for confirmation: Do you want to assign copyright to the MetaBrainz Foundation or retain your own copyright instead? Both options are perfectly acceptable.

Reply:
I'm happy with assigning copyright to the MetaBrainz Foundation, so we can keep it as is.